test/integration/scheduler_perf/config/pod-with-claim-reference.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  Name: test-dra---counter
```

Comment:
why counter? Also, shouldn't this use generateName? But probably not `Name` (uppercase N).

Reply:
I needed a special string that is accepted inside YAML.

I chose `counter` because its a running counter starting at zero inside the current operation. I chose `--counter` because `<counter>` was harder to use in JSON (gets encoded differently).

Using `generateName` defeats the purpose of having deterministic names. But why does `Name` work? Interesting. It looks like it doesn't matter. The Go field is properly populated.

Reply:
I'm replacing `Name` with `name`, of course - it's still the right spelling for YAML.